### PR TITLE
GSOC'17 blog post

### PR DESCRIPTION
**Pull request by guusdk**

### CODE REVIEW

**Comment by tfar** [comment]
I think before we go out and ask people for explicit project ideas for GSoC, we should at least have people we agree to serve as organisation admin and manage the whole thing. That is asking the open source projects to contribute ideas, making sure projects have teaser tasks, etc.

Feels a bit fishy to ask for ideas and later ending up having nobody to run this thing as org admin.

File: content/posts/blog/google-summer-of-code-2017.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Google Summer of Code 2017
+date: 2017-01-2272 11:57
+author: Guus
+blog_id: blog
+---
+
+As before, the [XSF](http://xmpp.org/) is considering to apply to this years [Google Summer of Code](https://summerofcode.withgoogle.com/). 
+
+The XSF is likely to act as an umbrella organisation for any XMPP-related project that wishes to join the GSoC. If you are a member of such a project and would like your project to be involved, get in touch!
```
Comment (tfar) [inline content/posts/blog/google-summer-of-code-2017.md]: This date looks a bit strange.

File: content/posts/blog/google-summer-of-code-2017.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Google Summer of Code 2017
+date: 2017-01-2272 11:57
```
Comment (tfar) [inline content/posts/blog/google-summer-of-code-2017.md]: This date looks a bit strange.

Comment (guusdk) [inline content/posts/blog/google-summer-of-code-2017.md]: Oops - will fix and squash.

**Comment by guusdk** [comment]
Sure. As far as I'm concerned we hit 'merge' only after we're ready. We might have an update for this text by then too.

**Comment by Kev** [comment]
@guusdk should check my tweaks, and if he's happy with that, I'm happy. I don't want to make this my first website merge and break anything.

**Comment by guusdk** [comment]
@guusdk is ecstatic (and not worried by a website breakage over a simple text change - famous last words, right?).

**Pull request merged**